tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +36,11 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
 def test_removing_not_found_returns_none():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == None
```

Comment:
Nice assert, we definitely want to confirm our return value is `None`. We generally also want to confirm that there weren't side affects or changes to the input that we didn't intend. What could we check about the vendor's inventory to confirm that its content didn’t change?

Reply:
assert len(vendor.inventory) == 3

Reply:
Definitely! Since knowing the length doesn't guarantee the individual elements are still what we expect, we might also check the elements of the inventory list.